packages/react/src/components/Pagination/next/__tests__/Pagination-test.js

```diff
@@ -0,0 +1,526 @@
+/**
```

Comment:
I believe this file was removed in https://github.com/carbon-design-system/carbon/pull/12115